src/solve.jl

```diff
@@ -0,0 +1,16 @@
+@inline function (\){T}(A::SMatrix{2,2,T,4}, b::SVector{2,T})
+    d = det(A)
+    @inbounds x = SVector( A[2,2]*b[1] - A[1,2]*b[2],
+                           A[1,1]*b[2] - A[2,1]*b[1] ) / d
+    return x
+end
+
+@inline function (\){T}(A::SMatrix{3,3,T,9}, b::SVector{3,T})
```

Comment:
As the package exists now, there's no nice way to overload this for all 3x3 `StaticMatrix` subtypes.  If you look at the code of `inv`, you'll see the unfortunate hardcoded trait-like workaround that we have so far.  What we really need is a proper size trait for fixed size dispatch.  In the meantime, I'd suggest copying the `inv` implementation, otherwise this won't work for `SArray`, `MMatrix`, `MArray`.  (We'd like to remove some of this zoo of types, but it's hard to achieve nice statically sized type names with current compiler capabilities.)